shared/menubar/chat.desktop.js

```diff
@@ -19,11 +16,19 @@ const ChatViewAll = ({onViewAll}: {onViewAll: () => void}) => (
   </Kb.Box2>
 )
 
-const ChatContainer = ({onViewAll}: {onViewAll: () => void}) => (
+type ChatContainerProps = {
+  onViewAll: () => void,
+  onSelectConversation: (ChatTypes.ConversationIDKey) => void,
+  conversations: Array<RemoteContainer.RemoteConvMeta>,
+}
+
+export default ({onViewAll, onSelectConversation, conversations}: ChatContainerProps) => (
   <Kb.Box2 direction="vertical" fullWidth={true} style={styles.chatContainer}>
-    <ChatRow name="one" />
-    <ChatRow name="two" />
-    <ChatRow name="three" />
+    {conversations.map(c => {
+      return (
+        <SmallTeam.SmallTeam key={c.conversationIDKey} {...c} onSelectConversation={() => onSelectConversation(c.conversationIDKey)} />
+      )
+    })}
```

Comment:
It seems to me it's better to do this mapping in the container (`mergeProps`). We really only need the `conversationIDKey` and an action derived from it here. So instead of iterating on `conversations`, we'd be iterating on `{conversationIDKey, onSelectConversation}` here, and the component would be dumber. We'd also be able to avoid having an action creator (`() => onSele...`) here.